client/e2e/pages/authenticated/dashboard/profile/page.spec.ts

```diff
@@ -6,39 +6,62 @@ import { DashboardPOM } from "@/e2e/poms/dashboard";
 import { ProfilePOM } from "@/e2e/poms/profile";
 // ☰ Enums
 import { UserRole } from "@/e2e/utils/enums";
+// 🥞 DB CRUD
+import { deleteUserRecord } from "@/e2e/utils/queries";
 // ℹ️ Environment variables
 import * as dotenv from "dotenv";
 dotenv.config({ path: "./e2e/.env.local" });
 
+// 📚 Declare a beforeAll hook that is executed once per worker process before all tests.
+// 🥞 Set DB for e2e login roles
+/*
+For "new user":
+-  delete record in the db so that on "new user" login the ID will have no app_role
+*/
+test.beforeAll(async () => {
+  try {
+    // 👤 delete new user: bc-cas-dev-three
+    await deleteUserRecord(process.env.E2E_NEW_USER_GUID as string);
```

Comment:
Further to our conversation about seeding/resetting, are there any tests where we need the new user to already be in the db? If not, maybe we can remove them from global so we don't have to add/delete them, and they can just start out not existing

Reply:
@BCerki there is no need for `new user to already be in the db`.  The profile update test will create the record for new user which is negated by the berforeAll hook.  If we have agreed to tread lightly in terms of implementing seeding/resetting with a watchful eye on resource hogging then I might leave the pages\ directory setup as is until further review of the seeding process.